### DISCUSSION TITLE
Possible to suppress diagnostics for specific errors?

### DISCUSSION THREAD

**Discussion by decimad** [clangd/clangd#1053] [Q&A] [answered]
Hey there,

I'm working on an unusual platform with a tailored gcc toolchain. It supports inline asm constraint expressions that stock gcc and clang does not know. Hence I get errors displayed such as "In included file: invalid output constraint '=d' in asmclang(asm_invalid_output_constraint)"
Is there a way to hide these errors? It appears the parser is able to recover from this error and continue processing the file.

Kind regards

**Comment by HighCommander4** [comment, marked as answer]
Yes, please have a look at https://clangd.llvm.org/config.html#suppress
